scripts/pre-commit.sh

```diff
@@ -107,6 +107,7 @@ function check-git-state() {
   for line in "${output}"; do
     echo "${line}"
   done
+  git diff
```

Comment:
Is this intentional?

Reply:
Yes, it's very helpful to figure out what has changed in the files.

Reply:
Example: https://travis-ci.org/kubernetes-sigs/federation-v2/builds/499900877#L1443